src/helpers/formatString.js

```diff
@@ -0,0 +1,9 @@
+import { isEmptyStr } from 'helpers/isEmptyStr'
+// Returns a formatted string
+// e.g) 'HOMO_SAPIENS' to 'Homo sapiens'
+
+export function formatString(key) {
+  const temp = key.toLowerCase().replace(/_/g, ' ')
+
+  return !isEmptyStr(key) ? temp.charAt(0).toUpperCase() + temp.slice(1) : null
+}
```

Comment:
I think another approach would be:

```suggestion
export function formatString([first = "", ...rest]) {
  return [first.toUpperCase(), ...rest].join('').replace(/_/g, ' ')
}
```